WolframLanguage.sublime-syntax

```diff
@@ -47,16 +47,33 @@ variables:
       Catalan|Degree|E|EulerGamma|Glaisher|GoldenAngle|GoldenRatio|I|Khinchin|MachinePrecision|Pi
     )\b
 
+  first_param_functional_functions: |-
+    (?x)
+    Apply|ArrayFilter|
+    BlockMap|
+    FixedPoint|FixedPointList|Fold|FoldList|FoldPair|FoldPairList|
+    Map|MapAt|MapIndexed|MapThread|
+    Nest|NestList|NestGraph|NestWhile|NestWhileList|
+    Scan|SequenceFold|SequenceFoldList
+
+  last_param_functional_functions: |-
+    (?x)
+    CountDistinctBy|LengthWhile|TakeWhile
+
+  parametic_functions: Table|Do|Sum|Product
```

Comment:
Why do you call them parametric functions? These are function with iterators.

Reply:
I guess you are right. But now I found a name which could be better: "[local variables](https://reference.wolfram.com/language/ref/SyntaxInformation.html)".

Reply:
Will be discussed in another PR.

Resolved.